src/Drupal/Commands/core/EntityDisplayCommands.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drush\Drupal\Commands\core;
+
+use Drush\Commands\DrushCommands;
+use Drush\Exceptions\UserAbortException;
+
+class EntityDisplayCommands extends DrushCommands
+{
+    /**
+     * Rebuild form and display configurations for entities and bundles.
+     *
+     * See Drupal Core: \Drupal\field\EntityDisplayRebuilder::rebuildEntityTypeDisplays
+     *
+     * @command entity:rebuild-displays
+     *
+     * @param string $entity_type_option Entity-type machine-name.
```

Comment:
Lets remove 'option' from both param names. They are arguments not options. Also we can remove dash in entity-type. And lets omit 'machine-name' entirely. User can figure that out from Examples.

Reply:
Inside the loops in the actual `rebuildDisplays`  method, there are multiple usages of $bundle and $entity_type variables. Example:
```
if ($bundle !== $bundle_arg && $bundle_arg !== 'all') {}
```
and
```
if ($entity_type !== $entity_type_arg && $entity_type_arg !== 'all') {}
```

There is a bit of a trade-off to rename the arguments or the variables inside the loops.
What is your opinion about this?

Reply:
[done] Also we can remove dash in entity-type.
[done] And lets omit 'machine-name' entirely. User can figure that out from Examples.